modules/table/src/integrationTest/java/org/apache/ignite/internal/threading/ItKvRecordApiThreadingTest.java

```diff
@@ -464,8 +465,8 @@ RecordContext<?> context() {
     }
 
     private enum CommonViewAsyncOperation {
-        QUERY_ASYNC(view -> view.queryAsync(null, null)),
-        QUERY_ASYNC_WITH_INDEX_NAME(view -> view.queryAsync(null, null, null)),
+        QUERY_ASYNC(view -> view.queryAsync(null)),
+        QUERY_ASYNC_WITH_INDEX_NAME(view -> view.queryAsync((Criteria) null, null)),
```

Comment:
User code will break in the same way as this test, let's avoid breaking changes.